apps/elasticsearch/vars/all/main.yml

```
@@ -0,0 +1,43 @@
+# -- Docker related configuration
+
+elasticsearch_image_name: "fundocker/openshift-elasticsearch"
+elasticsearch_image_tag: "6.6.2"
+
+# -- Node specific configuration
+
+elasticsearch_java_opts: "-Xms512m -Xmx512m"
```

Making it a list would be more convenient?

```suggestion
elasticsearch_java_opts:
  - "-Xms512m"
  - "-Xmx512m"
```
(you will need to use the `join` filter where it is used)